include/libtorrent/natpmp.hpp

```diff
@@ -79,8 +113,28 @@ struct TORRENT_EXTRA_EXPORT natpmp
 
 	void disable(error_code const& ec);
 
+	enum protocol_version
```

Comment:
I'm a fan of `enum class`. Then you can remove the `version_` and `opcode_` prefixes of the enum values too

Reply:
The reason I didn't use `enum class` for these is that they primarily serve as aliases for their integral values. If I used `enum class` I'd have to litter the code with casts to `int`.